client/blocks/reader-related-card-v2/docs/example.jsx

```diff
@@ -73,6 +83,9 @@ const moreOnWordPress = [
 			short_excerpt: "This presidential race has, to be blunt, a cluster bomb of attacks, lies and outright nonsense rhetoric that has astounded many and made people curse at their…",
 			canonical_image: {
 					uri: "https://freedomsandtruth.files.wordpress.com/2016/09/trump-rally-in-vegas-getty-640x480.jpg?w=720&quality=80&strip=info",
+			},
+			canonical_media: {
+					src: "https://freedomsandtruth.files.wordpress.com/2016/09/trump-rally-in-vegas-getty-640x480.jpg?w=720&quality=80&strip=info",
```

Comment:
> And not pull files from live sites
i actually think it would be best if all of our examples were `connect`ed to real objects.  That way we wouldn't run into issue like this where they get out of sync.